doc/sphinx/README.rst

```diff
@@ -0,0 +1,312 @@
+    This behaves like a generic admonition; see
+    http://docutils.sourceforge.net/docs/ref/rst/directives.html#generic-admonition
+    for more details.
+
+    Example::
+
+       .. example:: Adding a hint to a database
+
+          The following adds ``plus_comm`` to the ``plu`` database:
+
+          .. coqdoc::
+
+             Hint Resolve plus_comm : plu.
+
+``.. inference::`` A reST directive to format inference rules.
+    This also serves as a small illustration of the way to create new Sphinx
+    directives.
+
+    Usage::
+
+       .. inference:: name
+
+          newline-separated premisses
+          ------------------------
+          conclusion
+
+    Example::
+
+       .. inference:: Prod-Pro
+
+          \WTEG{T}{s}
+          s \in \Sort
+          \WTE{\Gamma::(x:T)}{U}{\Prop}
+          -----------------------------
+          \WTEG{\forall~x:T,U}{\Prop}
+
+``.. preamble::`` A reST directive for hidden math.
+    Mostly useful to let MathJax know about `\def`\ s and `\newcommand`\ s.
+
+    Example::
+
+       .. preamble::
+
+          \newcommand{\paren}[#1]{\left(#1\right)}
+
+Coq roles
+=========
+
+In addition to the objects and directives above, the ``coqrst`` Sphinx plugin defines the following roles:
+
+``:g:`` Coq code.
+    Use this for Gallina and Ltac snippets::
+
+       :g:`apply plus_comm; reflexivity`
+       :g:`Set Printing All.`
+       :g:`forall (x: t), P(x)`
+
+``:n:`` Any text using the notation syntax (``@id``, ``{+, …}``, etc.).
+    Use this to explain tactic equivalences.  For example, you might write
+    this::
+
+       :n:`generalize @term as @ident` is just like :n:`generalize @term`, but
+       it names the introduced hypothesis :token:`ident`.
+
+    Note that this example also uses ``:token:``.  That's because ``ident`` is
+    defined in the the Coq manual as a grammar production, and ``:token:``
+    creates a link to that.  When referring to a placeholder that happens to be
+    a grammar production, ``:token:`…``` is typically preferable to ``:n:`@…```.
+
+``:production:`` A Sphinx role to format grammar productions not included in a
+    `productionlist` directive.
+
+    Useful to informally introduce a production, as part of running text.
+
+    Example::
+
+       :production:`string` indicates a quoted string.
+
+    You're not likely to use this role very commonly; instead, use a
+    `production list
+    <http://www.sphinx-doc.org/en/stable/markup/para.html#directive-productionlist>`_
+    and reference its tokens using ``:token:`…```.
+
+Tips and tricks
+===============
+
+Abbreviations and macros
+------------------------
+
+Abbreviations and placeholders for specially-formatted names (like ``|Cic|``, ``|Coq|``, ``|CoqIDE|``, and ``|Gallina|``) are defined in a `separate file <doc/sphinx/replaces.rst>`_ included by most chapters of the manual.  Some useful LaTeX macros (like ``\alors``), are defined in `<doc/sphinx/replaces.rst>`_.
```

Comment:
Same problem with the links.
You can also list `|Ltac|` as part of the replacement patterns (it was introduced by @maximedenes when porting the Ltac chapter).
I also wondered why not all the chapters of the manual import the same two files. Should we enforce this?
And finally, is `\alors` a well-known LaTeX macro? To me it just sounds like a French word.